examples/ping-pong/src/main.rs

```diff
@@ -55,6 +55,7 @@ async fn main() {
 
     // This can be any string
     let radio_name: &str = "ping-pong";
+    let graphcast_namespace: &str = "";
```

Comment:
if we're setting the `testnet` value below do we need this var?

Reply:
nope! not sure why the precommit hook didn't catch this 🤔 but removed now